Games/types/Mafia/roles/cards/OrangeGiver.js

```diff
@@ -1,15 +1,23 @@
 const Card = require("../../Card");
 const { PRIORITY_ITEM_GIVER_DEFAULT } = require("../../const/Priority");
+const { EXCLUSIVE_PRIORITY_HOT_SPRINGS } = require("../../const/MeetingExclusivePriority");
 
 module.exports = class OrangeGiver extends Card {
 
     constructor(role) {
         super(role);
 
         this.meetings = {
+            "Hot Springs": {
+                states: ["Night"],
+                flags: ["exclusive", "group", "speech", "anonymous"],
```

Comment:
same as jailor, it needs to be exclusive or there is the behaviour that
- when capy is alone, meeting is not exclusive (i.e. can attend parties)
- when capy is not alone, meeting is exclusive